benchmarks/cpp/tensorexpr/bench_signed_log1p.cpp

```diff
@@ -22,7 +23,7 @@ class SignedLog1pBench : public benchmark::Fixture {
   }
 
   void TearDown(benchmark::State& state) override {
-    TORCH_CHECK(at::allclose(ref_, output_));
+    TORCH_CHECK(at::allclose(ref_, output_, 1e-3, 1e-3));
```

Comment:
The test fails when `allclose` uses the default threshold as before. And that happens only when `log_vml` is enabled (I just confirmed that again).

We will definitely have to debug the numerics before enabling `log_vml` as the default.

Reply:
Hmm.  So I have a test for log_vml (I think it’s in test/cpp/tensorexpr) that sweeps large swaths of floating point numbers to check for accuracy.  My recollection was that it was either bit-accurate or within 1 ulp if VML (it depended on the platform; on avx512 machines vml uses a slightly different algorithm). We should do a similar check for log1p.  It would be unfortunate if log(1+x) (our implementation) is different from log1p(x)